.drone.yml

```diff
@@ -1,53 +0,0 @@
-kind: pipeline
-name: default
-
-steps:
-- name: ci
-  image: rancher/dapper:v0.6.0
-  commands:
-  - dapper ci
-  volumes:
-    - name: socket
-      path: /var/run/docker.sock
```

Comment:
It looks like `ci` is expected to run on all events. Could you add a workflow that runs `dapper ci` in the event of `pull request` and `merge`?